packages/cloud-alert/cypress/support/commands.ts

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Can we delete this file?

Reply:
@vuhuucuong the file come from the template. I think it would be nice if we let there for full structure of cypress project